src/theme/variables/edgeDark.ts

```diff
@@ -169,10 +169,10 @@ export const edgeDark: Theme = {
 
   // Header
   headerIcon: edgeMark,
-  headerBackground: [palette.darkestNavy, `${palette.darkAqua}cc`],
+  headerBackground: [`#000000`, `#00000000`],
```

Comment:
Do we have these in the palette? The second one could probably be `transparent`. Otherwise, use regular quotes instead of backtick quotes.